identifiers/country-et/README.md

```diff
@@ -0,0 +1,11 @@
+# OCD Division Identifier
+
+This folder contains the following Ethiopian administrative divisions :
+
+## State
+* Regional states, represented as *state* and city administrations represented as *administration*
+
+The OCD identifiers are created based on the ISO 3166-2 code [(source)](https://en.wikipedia.org/wiki/ISO_3166-2:ET)
+
+## Constituencies
+* Constituencies represented as *ed*, sourced from [hopr.gov.et](http://hopr.gov.et/)
```

Comment:
Is there a more specific page to link to for the source of the constituencies?  The website is down currently so browsing through archive.org is slow, but I'm having trouble finding the correct page.